Misc/NEWS.d/next/Core and Builtins/2022-05-21-23-21-37.gh-issue-93065.5I18WC.rst

```diff
@@ -0,0 +1,5 @@
+Fix contextvars HAMT implementation to handle iteration over deep trees.
+
+The bug was discovered and fixed by Eli Libman. See [1] for more details.
+
+[1] https://github.com/MagicStack/immutables/issues/84
```

Comment:
This doesn't look great in the actual change log later:
<img width="814" alt="Screen Shot 2022-05-23 at 19 38 32" src="https://user-images.githubusercontent.com/55281/169881776-93025663-503c-495f-afb2-1e46279db242.png">

I'll change it to an inline link and land.

Reply:
After:
<img width="797" alt="Screen Shot 2022-05-23 at 20 19 33" src="https://user-images.githubusercontent.com/55281/169882554-09403f96-eb62-4b73-bf54-3fe5fe74bf61.png">